talentsearch/.env.example

```diff
@@ -2,7 +2,7 @@ APP_NAME="Talent Search"
 APP_ENV=local
 APP_KEY=
 APP_DEBUG=true
-TALENTSEARCH_APP_URL="http://localhost:8000"
+TALENTSEARCH_APP_URL="http://localhost:8000/talentsearch"
```

Comment:
You've set this to `/talentsearch`, but didn't change the infrastructure htaccess file, or the talentsearch/routes/web file, or use this base url in the talentsearch client-side router. We should either leave this as a site-wide landing page for now, or go all the way.